examples/hacktoberfest-diagram/index.html

```diff
@@ -0,0 +1,69 @@
+<!--
+Copyright 2020 Bonitasoft S.A.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>BPMN Visualization - Hacktoberfest diagram</title>
+    <link rel="shortcut icon" href="../favicon.ico">
+    <link rel="stylesheet" href="../static/css/spectre.min.css" type="text/css">
+    <link rel="stylesheet" href="../static/css/icons.min.css" type="text/css">
+    <link rel="stylesheet" href="../static/css/main.css" type="text/css">
+</head>
+<body>
+    <!-- load global settings -->
+    <script src="../../demo/0.3.0/static/js/configureMxGraphGlobals.js"></script>
+    <!-- load mxGraph client library -->
+    <script src="../../demo/0.3.0/static/js/mxClient.min.js"></script>
+    <!-- load the example -->
+    <script src="./hacktoberfest-diagram.js" type="module"></script>
+
+    <header class="navbar bg-dark p-2">
+        <section class="navbar-section ml-2">
+            <img src="../static/img/logo_64x64.ico" alt="logo" class="logo">
+            <span class="h3 text-light ml-2 lib-name-text">BPMN Visualization Examples</span>
+        </section>
+        <section class="navbar-section">
+            <a href="https://process-analytics.github.io/bpmn-visualization-js/" class="mr-2 btn btn-secondary">Docs</a>
+            <div class="dropdown mr-2">
+                <a href="#" class="btn btn-primary dropdown-toggle" tabindex="0">
+                    GitHub <span class="icon icon-caret icon-in-the-middle"></span>
+                </a>
+                <ul class="menu">
+                    <li><a href="https://github.com/process-analytics/bpmn-visualization-js" class="mr-2 btn btn-link">Library</a></li>
+                    <li><a href="https://github.com/process-analytics/bpmn-visualization-examples" class="mr-2 btn btn-link">Examples</a></li>
+                </ul>
+            </div>
+        </section>
+    </header>
+    <section class="container col-10 flex-centered mt-2">
+        <section class="col-12 mt-2">
+            <div class="col-12 mb-2">
```

Comment:
:question: I suggest to add an explanation for people coming here because they land to this page during Hacktober Fest.
Remind them what the can do: enter their project name, take a screenshot of the diagram using the theme they want, advertise about their project with the diagram

Reply:
Done